web/frontend/src/components/slam-2d-render.vue

```diff
@@ -348,10 +346,16 @@ const updatePointCloud = (pointcloud: Uint8Array) => {
     axesNeg
   );
 
-  if (props.pose !== undefined) {
```

Comment:
[q] This removal is because we don't need to check the pose as it will always exist (since the marker is created at the beginning). Correct?